app/controllers/photos_controller.rb

```diff
@@ -9,21 +8,26 @@ def index
 
   # GET /photos/1 or /photos/1.json
   def show
+    authorize @photo
   end
 
   # GET /photos/new
   def new
     @photo = Photo.new
+    @user = current_user
+    authorize @user, :new?
   end
 
   # GET /photos/1/edit
   def edit
+    authorize @photo
   end
 
   # POST /photos or /photos.json
   def create
     @photo = Photo.new(photo_params)
     @photo.owner = current_user
+    authorize @photo.owner, :new?
```

Comment:
do we need to call the User#new policy here? I think we should be able to just call `authorize @photo`

Reply:
resolved